templates/include/telemetry/mainPayload.1.schema.json

```diff
@@ -147,6 +147,10 @@
               "maxItems": 2
             }
           }
+        },
+        "isFromTerminatorWatchdog": {
+          "description": "True if the late writes began in nsTerminator, false otherwise",
+          "type": "boolean"
```

Comment:
I checked the values incoming from nightly on 2020-04-30, and they seem to be integer values ([query](https://sql.telemetry.mozilla.org/queries/70617/source)). Does this align with your understanding of the implementation? This should be set to `integer` to reflect the values of data that is flowing in now.